src/Microsoft.AspNet.OData.Shared/Builder/ODataModelBuilder.cs

```diff
@@ -28,13 +28,14 @@ public class ODataModelBuilder
 
         private Version _dataServiceVersion;
         private Version _maxDataServiceVersion;
+        private string _namespace;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ODataModelBuilder"/> class.
         /// </summary>
         public ODataModelBuilder()
```

Comment:
>ODataModelBuilder [](start = 15, length = 17)

For code clarity, should initialize HasDefaultNamespace to false in the constructor.